frontend/packages/kubevirt-plugin/integration-tests/tests/utils/utils.ts

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable no-await-in-loop, no-console, no-underscore-dangle */
```

Comment:
This (and other utils if appropriate) could be moved under `packages/console-shared/src/test-utils`.

Reply:
Done.